app/views/works/_work_header_navigation.html.erb

```diff
@@ -1,6 +1,9 @@
 <!-- BEGIN navigation -->
 <h3 class="landmark heading"><%= ts("Actions") %></h3>
 <ul class="work navigation actions" role="menu">
+  <% if ( logged_in_as_admin? || permit?("tag_wrangler") ) %>
+    <li class="reindex"><%= link_to ts("Reindex Work"), reindex_work_path(@work) %></li>
```

Comment:
After the style rules for `li.relationships a` in https://github.com/otwcode/otwarchive/blob/master/public/stylesheets/site/2.0/10-types-groups.css#L168, let's do

```
.reindex a {
  border-bottom: 3px solid #900;
}

to give the button a red bottom border, like the teal one on admin actions. I don't love using red on buttons, but it's good enough for now.